packages/kit/src/core/postbuild/prerender.js

```diff
@@ -112,7 +112,10 @@ async function prerender({ out, manifest_path, metadata, verbose, env }) {
 					? `${path} does not begin with \`base\`, which is configured in \`paths.base\` and can be imported from \`$app/paths\` - see https://kit.svelte.dev/docs/configuration#paths for more info`
 					: path;
 
-			return `${status} ${message}${referrer ? ` (${referenceType} from ${referrer})` : ''}`;
+			return (
+				`${status} ${message}${referrer ? ` (${referenceType} from ${referrer})` : ''}` +
+				`\nTo suppress or handle this error, implement \`handleHttpError\` in https://kit.svelte.dev/docs/configuration#prerender`
```

Comment:
I think that if the user implements `handleHttpError` this will be the `message` passed to that function and it might be undesirable to include  "To suppress or handle this error..." there as you'll be getting that message even when you do implement the function

Reply:
Well now we're full-circle, ha -- what's the solution to the issue if not this?

Reply:
Actually, looking at the signature for the handler, I'm not sure this message would be undesirable -- we pass through the parsed information they need to do actual handling, so hopefully they're not trying to parse the message or something silly like that...

Reply:
I tend to agree with @tcc-sejohnson - the message is desirable IMO

Reply:
Ben's right, it's weird for this text to show up if the handler was implemented. Whether or not it's _likely_ that someone will use `message` in their own handlers is separate to the question of whether the message is correct (but the whole reason it's provided is so that you _can_ use it)